### PR TITLE
Add links to newly added notebooks in the website #756

### PR DESCRIPTION
**Pull request by olaoluwasalami**
@sonichi @thinkall @BeibinLi 

add a new notebook to website

## Related issue number
#756 

## Checks

- [x] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Review by sonichi — COMMENTED** [review]
@BeibinLi @qingyun-wu where are the other multimodal notebook examples?
We have two webpages that contain the same set of examples. People often forget to update them both. What shall we do?

**Comment by olaoluwasalami** [comment]
> @BeibinLi @qingyun-wu where are the other multimodal notebook examples? We have two webpages that contain the same set of examples. People often forget to update them both. What shall we do?

I can handle this webpages going forward @sonichi

File: website/docs/Examples/AgentChat.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Links to notebook examples:
    - Task Solving with Langchain Provided Tools as Functions - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_function_call.ipynb)
    - **RAG**: Group Chat with Retrieval Augmented Generation (with 5 group member agents and 1 manager agent) - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_groupchat_RAG.ipynb)
    - In-depth Guide to OpenAI Utility Functions - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/oai_openai_utils.ipynb)
+   - **DALLE**: Agent Chat with Multimodal Models: DALLE  and GPT-4V - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_dalle_and_gpt4v.ipynb)
```
Comment (qingyun-wu) [inline website/docs/Examples/AgentChat.md:39]:
```suggestion
   - **Multimodal**: Agent Chat with Multimodal Models: DALLE  and GPT-4V - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_dalle_and_gpt4v.ipynb)
```

File: website/docs/Use-Cases/agent_chat.md [hunk 2]
```diff
@@ -110,6 +110,7 @@ The figure below shows six examples of applications built using AutoGen.
    - Task Solving with Langchain Provided Tools as Functions - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_langchain.ipynb)
    - **RAG**: Group Chat with Retrieval Augmented Generation (with 5 group member agents and 1 manager agent) - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_groupchat_RAG.ipynb)
    - In-depth Guide to OpenAI Utility Functions - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/oai_openai_utils.ipynb)
+   - **DALLE**: Agent Chat with Multimodal Models: DALLE  and GPT-4V - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_dalle_and_gpt4v.ipynb)
```
Comment (qingyun-wu) [inline website/docs/Use-Cases/agent_chat.md:113]:
```suggestion
   - **Multimodal**: Agent Chat with Multimodal Models: DALLE  and GPT-4V - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_dalle_and_gpt4v.ipynb)
```

**Comment by qingyun-wu** [comment]
> > @BeibinLi @qingyun-wu where are the other multimodal notebook examples? We have two webpages that contain the same set of examples. People often forget to update them both. What shall we do?
> 
> I can handle this webpages going forward @sonichi

@olaoluwasalami thanks for being willing to help on this. Going forward we may want to consider re-factoring the website so that the links to notebooks do not appear on two pages. @sonichi

**Comment by olaoluwasalami** [comment]
PR updated @qingyun-wu

**Review by qingyun-wu — APPROVED** [review]

File: website/docs/Examples/AgentChat.md [hunk 3]
```diff
@@ -36,6 +36,7 @@ Links to notebook examples:
    - Task Solving with Langchain Provided Tools as Functions - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_function_call.ipynb)
    - **RAG**: Group Chat with Retrieval Augmented Generation (with 5 group member agents and 1 manager agent) - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_groupchat_RAG.ipynb)
    - In-depth Guide to OpenAI Utility Functions - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/oai_openai_utils.ipynb)
+   - **Multimodal**: Agent Chat with Multimodal Models: DALLE  and GPT-4V - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_dalle_and_gpt4v.ipynb)
```
Comment (qingyun-wu) [inline website/docs/Examples/AgentChat.md:39]: How about adding a new **Multimodal Agent** subsection and include all the relevant notebooks below: 
   1. Multimodal Agent Chat with DALLE  and GPT-4V - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_dalle_and_gpt4v.ipynb)
   2. Multimodal Agent Chat with Llava - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_lmm_llava.ipynb)
   3. Multimodal Agent Chat with GPT-4V - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_lmm_gpt-4v.ipynb)

Comment (olaoluwasalami) [reply website/docs/Examples/AgentChat.md]: Updated @qingyun-wu

**Pull request merged**